src/os/bluestore/BlueFS.cc

```diff
@@ -609,8 +614,13 @@ int BlueFS::_replay(bool noop)
       return -EIO;
     }
     assert(seq == t.seq);
-    dout(10) << __func__ << " 0x" << std::hex << pos << std::dec
-             << ": " << t << dendl;
+    if (unlikely(to_stdout)) {
+      std::cout << " 0x" << std::hex << pos << std::dec
+                << ": " << t << std::endl;
+    } else {
+      dout(10) << __func__ << " 0x" << std::hex << pos << std::dec
+               << ": " << t << dendl;
```

Comment:
let's leave the dout unconditional so it's just if (to_stdout) std::cout << ...

Reply:
ok